esp-hal-common/src/lib.rs

```diff
@@ -113,6 +113,8 @@ pub mod peripheral;
 pub mod prelude;
 #[cfg(radio)]
 pub mod radio;
+#[cfg(any(hmac, sha))]
+pub mod reg_access;
```

Comment:
nit-pick but this doesn't need to be `pub`? Just because it will appear in the docs this way

Reply:
+1